starwars/src/components/Char.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+
+function Char(props) {
+  console.log('props', props);
+  return (
+  <div>
+    <button>{props.nameprops.name}</button>;
```

Comment:
So here, you are getting `nameprops` passed in from props. `nameprops` is really an object holding all the key/value pairs of each character (e.g. name, gender, height, etc). Meaning you can add the rest of the character stats in here using the same logic. This is what I showed you in our 1:1; if you don't remember how I got that, let me know, I'll go over it with you again.

Like I suggested before, if you change your `nameprops` naming convention to something like `character`, I think it will be more intuitive to you what that data is and what you can do with it.